diff -up spectacle-20.03.80/src/QuickEditor/QuickEditor.cpp.omv~ spectacle-20.03.80/src/QuickEditor/QuickEditor.cpp
--- spectacle-20.03.80/src/QuickEditor/QuickEditor.cpp.omv~	2020-03-23 14:40:51.095142633 +0100
+++ spectacle-20.03.80/src/QuickEditor/QuickEditor.cpp	2020-03-23 14:40:59.360094071 +0100
@@ -22,6 +22,7 @@
 #include <KWayland/Client/surface.h>
 #include <QGuiApplication>
 #include <QScreen>
+#include <QPainterPath>
 #include <QtCore/qmath.h>
 
 #include "QuickEditor.h"
